kedro-datasets/kedro_datasets/polars/ipc_dataset.py

```diff
@@ -0,0 +1 @@
+"""TODO"""
```

Comment:
I'd suggest removing this file if you're not adding this dataset with this PR. And the same for the `polars/parquet_dataset`

Reply:
Done